blocks/bricks/__init__.py

```diff
@@ -464,6 +464,10 @@ def apply(self, input_):
             child_input = output
         return output
 
+    @apply.property('outputs')
```

Comment:
That's right, but your should remove `outputs` from the `@application` declaration. It would be also nice to permit multiple input in the same way as you did with outputs.